Dockerfile

```diff
@@ -41,11 +41,13 @@ RUN ln -fs /usr/lib/x86_64-linux-gnu/libldap.so /usr/lib/ \
 	&& docker-php-ext-install ldap \
 	&& a2enmod rewrite ldap headers
 
-# Install nodejs
-RUN curl -sL https://deb.nodesource.com/setup_4.x | bash - \
-	&& apt-get install -y nodejs \
+# Install nodejs-legacy, npm and karma
+RUN apt-get update && apt-get install -y nodejs-legacy npm \
```

Comment:
`apt-get update` is required, otherwise `nodejs-legacy` and `npm` and can't be found.